### DISCUSSION TITLE
Request for LaTeX Support

### DISCUSSION THREAD

**Discussion by ray-math** [mvdkwast/obsidian-copy-as-html#50] [Ideas]
As an Obsidian user, I wish to express the significant difficulty caused by the lack of LaTeX support on this platform. Currently, Obsidian plugin does not support the copying of LaTeX syntax, which results in a deficiency in accurately and clearly conveying mathematical expressions and scientific documentation. There is a demand among many users, including myself, to efficiently represent complex equations and data using LaTeX. Therefore, I earnestly request the addition of LaTeX support in the Obsidian platform. This feature would enhance user experience and be an invaluable tool, especially for those working in mathematics, science, and engineering fields.

**Comment by mvdkwast** [comment]
Hi @ray-math, I entirely agree about the usefulness of LaTeX :). This has also been asked before (#11), and I even did a POC implementation (https://github.com/mvdkwast/obsidian-copy-as-html/tree/feature/mathjax), but that was never merged because :

- It adds a big dependency to the plugin : the code that renders LaTeX to HTML
- the quality was not optimal IIRC
- there was only one person requesting the feature

So, still not sure how to do this well, since my time is pretty limited these days.

Also, I'm just a plugin developer, I cannot do anything about the "Obsidian platform".
